app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -52,8 +52,8 @@ You can apply for a tuition fee loan to cover the cost of your postgraduate teac
 
 The maximum tuition fee loan for postgraduate teacher training for a UK applicant is:
 
-* £9,250 per year for a full-time course
-* £6,935 per year for a part-time course
+* $fees_pgitt_domesticfulltime$ per year for a full-time course
```

Comment:
@Sarah-DfE These two variables are about loans rather than fees so I think I should create a new variable name in the `finance.yml` file , which is where the maintenance loan details are held, and replace these references - do you agree?

Reply:
Makes sense to me!

Reply:
New variables added to `finance.yml`- resolving this conversation